SPECS-EXTENDED/containernetworking-plugins/containernetworking-plugins.spec

```
@@ -69,87 +78,14 @@ containers, along with a number of supported plugins. CNI concerns itself
-Provides: golang(%{import_path}/pkg/ipam) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/ns) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/skel) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/testutils) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/types) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/types/020) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/types/current) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/utils) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/utils/hwaddr) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/utils/sysctl) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/version) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/version/legacy_examples) = %{version}-%{release}
-Provides: golang(%{import_path}/pkg/version/testhelpers) = %{version}-%{release}
-Provides: golang(%{import_path}/plugins/ipam/host-local/backend) = %{version}-%{release}
-Provides: golang(%{import_path}/plugins/ipam/host-local/backend/allocator) = %{version}-%{release}
-Provides: golang(%{import_path}/plugins/ipam/host-local/backend/disk) = %{version}-%{release}
-Provides: golang(%{import_path}/plugins/ipam/host-local/backend/testing) = %{version}-%{release}
-Provides: golang(%{import_path}/plugins/test/noop/debug) = %{version}-%{release}
-
-%description devel
-This package contains library source intended for
-building other packages which use import path with
-%{import_path} prefix.
-%endif
-
-%if 0%{?with_unit_test} && 0%{?with_devel}
-%package unit-test-devel
-Summary: Unit tests for %{name} package
-%if 0%{?with_check}
-%endif
-
-Requires: %{name}-devel = %{version}-%{release}
-
-%if 0%{?with_check} && ! 0%{?with_bundled}
-BuildRequires: golang(github.com/d2g/dhcp4)
-BuildRequires: golang(github.com/onsi/ginkgo)
-BuildRequires: golang(github.com/onsi/ginkgo/config)
-BuildRequires: golang(github.com/onsi/ginkgo/extensions/table)
-BuildRequires: golang(github.com/onsi/gomega)
-BuildRequires: golang(github.com/onsi/gomega/gbytes)
-BuildRequires: golang(github.com/onsi/gomega/gexec)
-BuildRequires: golang(github.com/vishvananda/netlink/nl)
-%endif
-
-Requires: golang(github.com/d2g/dhcp4)
-Requires: golang(github.com/onsi/ginkgo)
-Requires: golang(github.com/onsi/ginkgo/config)
-Requires: golang(github.com/onsi/ginkgo/extensions/table)
-Requires: golang(github.com/onsi/gomega)
-Requires: golang(github.com/onsi/gomega/gbytes)
-Requires: golang(github.com/onsi/gomega/gexec)
-Requires: golang(github.com/vishvananda/netlink/nl)
-
-%description unit-test-devel
-This package contains unit tests for project
-providing packages with %{import_path} prefix.
-%endif
 
 %prep
 %autosetup -Sgit -n %{repo}-%{built_tag_strip}
 rm -rf plugins/main/windows
 
+# Use correct paths in cni-dhcp unitfiles
+sed -i 's/\/opt\/cni\/bin/\%{_prefix}\/libexec\/cni/' plugins/ipam/dhcp/systemd/cni-dhcp.service
```

Suggestion: no need to escape `/`, if you use a different separator:
```suggestion
sed -i 's|/opt/cni\/bin|\%{_prefix}/libexec/cni|' plugins/ipam/dhcp/systemd/cni-dhcp.service
```

---

Updated.